src/js/providers/html5.js

```diff
@@ -294,9 +297,9 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
     let dvrPosition = null;
     let dvrUpdatedTime = 0;
 
-    this.isSDK = !!_playerConfig.sdkplatform;
```

Comment:
is this not used anywhere?

Reply:
Nope. Spoke with @karimMourra and searched all projects for `isSDK`. html5.js is not used to handle playback, and if it was nothing in the player uses this property.